### PR TITLE
fix the date calculation.

### PR DESCRIPTION
**Pull request by paule96**
The changes are not tested. Please wait bevor merge this.

### CODE REVIEW

File: src/MeetupToMarkdownConverter/Meetup/Helper.cs [hunk 1]
```diff
@@ -5,14 +5,23 @@ namespace MeetupToMarkdownConverter.Meetup
     using System.Text;
     internal class Helper
     {
-        #pragma warning disable SA1008 // StyleCop does not knor Tuples
+        public static DateTime FromUnixTime(long unixTime)
+        {
+            return epoch.AddSeconds(unixTime);
```
Comment (paule96) [inline src/MeetupToMarkdownConverter/Meetup/Helper.cs]: maybe this are millisecounds

**Comment by paule96** [comment]
In the heade of [this](https://raw.githubusercontent.com/dddotnet/dddotnet.github.io/master/_posts/2018-12-12-weihnachtstreffen---deine-10-minuten-ruhm.markdown) you see the time format.

Check this double. 🐱‍👓

**Comment by paule96** [comment]
@kirkone @oliverguhr please review my changes. I now have test this local.

The current output is the following

```markdown
---
layout: postnew
title: "Weihnachtstreffen - Deine 10 Minuten Ruhm"
categories: treffen
date: 2018-12-12T18:00:00.000Z
speaker: "siehe Meetup"
place: Develappers GmbH
placelink: https://maps.google.com/maps?f=q&hl=en&q=K%C3%B6nigsbr%C3%BCckerstr.+76%2C+Dresden%2C+de
---

## Weihnachtstreffen - Deine 10 Minuten Ruhm
<p>Das letzte Treffen in des Jahres steht wie immer unter dem Motto “Deine 10 Minuten Ruhm”.</p> <p>Am 12. Dezember findet ab 18 Uhr unser traditionelles Weihnachtstreffen statt. In lockerer Atmosphäre stehen nicht einzelne Sprecher, sondern die Gemeinschaft der Usergroup im Mittelpunkt. Denn bei der Veranstaltung kann jeder einen Vortrag von maximal 10 Minuten Länge über ein Thema das ihm am Herzen liegt halten.</p> 

## Ort und Zeit
Wir treffen uns am 12. December 2018 um 18:00 Uhr bei [Develappers GmbH](https://maps.google.com/maps?f=q&hl=en&q=K%C3%B6nigsbr%C3%BCckerstr.+76%2C+Dresden%2C+de).  
Die Gästeliste wird auf [Meetup](https://www.meetup.com/NET-User-Group-Dresden/events/256621880/) verwaltet.
```

the new method create this markdown:

```markdown

---
layout: postnew
title: "Weihnachtstreffen - Deine 10 Minuten Ruhm"
categories: treffen
date: 2018-12-12T17:00:00.000Z
speaker: "siehe Meetup"
place: Develappers GmbH
placelink: https://maps.google.com/maps?f=q&hl=en&q=K%C3%B6nigsbr%C3%BCckerstr.+76%2C+Dresden%2C+de
---

## Weihnachtstreffen - Deine 10 Minuten Ruhm
<p>Das letzte Treffen in des Jahres steht wie immer unter dem Motto “Deine 10 Minuten Ruhm”.</p> <p>Am 12. Dezember findet ab 18 Uhr unser traditionelles Weihnachtstreffen statt. In lockerer Atmosphäre stehen nicht einzelne Sprecher, sondern die Gemeinschaft der Usergroup im Mittelpunkt. Denn bei der Veranstaltung kann jeder einen Vortrag von maximal 10 Minuten Länge über ein Thema das ihm am Herzen liegt halten.</p> 

## Ort und Zeit
Wir treffen uns am 12. Dezember 2018 um 18:00 Uhr bei [Develappers GmbH](https://maps.google.com/maps?f=q&hl=en&q=K%C3%B6nigsbr%C3%BCckerstr.+76%2C+Dresden%2C+de).  
Die Gästeliste wird auf [Meetup](https://www.meetup.com/NET-User-Group-Dresden/events/256621880/) verwaltet.


```

File: src/MeetupToMarkdownConverter/Meetup/Helper.cs [hunk 2]
```diff
@@ -5,14 +5,23 @@ namespace MeetupToMarkdownConverter.Meetup
...
         internal static (string markdown, string filename) ConvertEventToMarkdown(Event meetup)
-        #pragma warning restore SA1008 // StyleCop does not knor Tuples
+#pragma warning restore SA1008 // StyleCop does not knor Tuples
         {
             StringBuilder markdown = new StringBuilder();
 
             var germanTimeZone = TimeZoneInfo.FindSystemTimeZoneById("W. Europe Standard Time");
-            var date = TimeZoneInfo.ConvertTimeFromUtc(new DateTime((meetup.Time * 10000) + 621355968000000000), germanTimeZone);
+
+            var date = FromUnixTime(meetup.Time).ToUniversalTime();
```
Comment (oliverguhr) [inline src/MeetupToMarkdownConverter/Meetup/Helper.cs:23]: looks way better now :)

**Review by oliverguhr — APPROVED** [review]
Nice fix, thank you.

**Pull request merged**